src/actors/symbolication.rs

```diff
@@ -498,33 +532,10 @@ impl SymCacheLookup {
                             .and_then(|result| {
                                 result
                                     .and_then(|symcache| match symcache.parse()? {
-                                        Some(_) => Ok((Some(symcache), DebugFileStatus::Found)),
-                                        None => {
-                                            Ok((Some(symcache), DebugFileStatus::MissingDebugFile))
-                                        }
-                                    })
-                                    .or_else(|e| {
-                                        let status = match e.kind() {
-                                            SymCacheErrorKind::Fetching => {
-                                                DebugFileStatus::FetchingFailed
-                                            }
-
-                                            // Timeouts of object downloads are caught by
-                                            // FetchingFailed
-                                            SymCacheErrorKind::Timeout => DebugFileStatus::TooLarge,
-
-                                            SymCacheErrorKind::ObjectParsing => {
-                                                DebugFileStatus::MalformedDebugFile
-                                            }
-
-                                            _ => {
-                                                capture_fail(&*e);
-                                                DebugFileStatus::Other
-                                            }
-                                        };
-
-                                        Ok((None, status))
+                                        Some(_) => Ok((Some(symcache), ObjectFileStatus::Found)),
+                                        None => Ok((Some(symcache), ObjectFileStatus::MissingFile)),
                                     })
+                                    .or_else(|e| Ok((None, symcache_error_to_status(&e))))
```

Comment:
This would constrain us to only sending `Found` when a symcache is present. My assumption is that we will extend this set of states to also include `Breakpad` and other variants where we have a symcache but still want to return a variety of statuses